calendar-app/Components/Footer.js

```diff
@@ -7,8 +7,6 @@ import GroupScreen from './GroupScreen';
 import HomeScreen from './HomeScreen';
 import CreateEventScreen from './CreateEventScreen';
 
-
-
```

Comment:
You should remove these spaces

Reply:
There were 2 extra spaces that I removed, you might have understood it incorrectly.